### ISSUE TITLE
Unable to parse 2BEG.cif

### ISSUE THREAD

**Issue by JoaoRodrigues** [PDB-REDO/libcifpp#18]
Reporting an issue with `libcifpp` when reading a mmCIF file downloaded from the wwPDB: [2BEG](https://www.rcsb.org/structure/2BEG). I get the following error when trying to use DSSP on this file:

```
  Missing residue for atom LEU A:17 N
  Missing residue for atom LEU A:17 CA
  Missing residue for atom LEU A:17 C
  Missing residue for atom LEU A:17 O
  Missing residue for atom LEU A:17 CB
  Missing residue for atom LEU A:17 CG
  Missing residue for atom LEU A:17 CD1
  Missing residue for atom LEU A:17 CD2
  Missing residue for atom VAL A:18 N
  Missing residue for atom VAL A:18 CA
  Missing residue for atom VAL A:18 C
  Missing residue for atom VAL A:18 O
  Missing residue for atom VAL A:18 CB
  Missing residue for atom VAL A:18 CG1
  Missing residue for atom VAL A:18 CG2
```

This goes on complaining for every single atom in the structure (900 heavy atoms per model, it's an NMR structure). Any pointers on what the issue might be? I can read `1GGR`, which also an NMR structure just fine.

This is currently blocking https://github.com/biopython/biopython/pull/3775.

**Comment by mhekkel** [comment]
Yup, confirmed. This is a bug in libcifpp. I'll try to release a fix asap.

Thanks for reporting.

**Comment by mhekkel** [comment]
I've released version 4.2.2 which should fix this particular problem. Could you please have a look and let me know if this fixes your issue?

**Comment by JoaoRodrigues** [comment]
Fixed asap indeed, thanks!
